internal/manifest/v1alpha/examples/slo_composite.go

```diff
@@ -0,0 +1,103 @@
+package v1alphaExamples
+
+import (
+	"fmt"
+	"time"
+
+	v1alphaSLO "github.com/nobl9/nobl9-go/manifest/v1alpha/slo"
+	"github.com/nobl9/nobl9-go/sdk"
+)
+
+type sloCompositeExample struct {
+	sloBaseExample
+}
+
+func (s sloCompositeExample) GetObject() any {
+	return s.Slo()
+}
+
+func (s sloCompositeExample) GetVariant() string {
+	return "composite-slo"
+}
+
+func (s sloCompositeExample) GetSubVariant() string {
+	return s.String()
+}
+
+func (s sloCompositeExample) GetYAMLComments() []string {
+	return []string{
+		fmt.Sprintf("Composite SLO"),
+		fmt.Sprintf("Budgeting method: %s", s.BudgetingMethod),
+		fmt.Sprintf("Time window type: %s", s.TimeWindowType),
+	}
+}
+
+func (s sloCompositeExample) String() string {
+	return fmt.Sprintf(
+		"Composite SLO using %s budgeting method and %s time window",
+		s.BudgetingMethod,
+		s.TimeWindowType,
+	)
+}
+
+func (s sloCompositeExample) Slo() v1alphaSLO.SLO {
```

Comment:
Let's stick to the conventional SLO naming, i.e. all capital letters:

```suggestion
func (s sloCompositeExample) SLO() v1alphaSLO.SLO {
```